payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherActivity.kt

```diff
@@ -179,9 +180,8 @@ internal class GooglePayLauncherActivity : AppCompatActivity() {
         finish()
     }
 
-    private fun disableAnimations() {
-        // this is a transparent Activity so we want to disable animations
-        overridePendingTransition(0, 0)
+    private fun setFadeAnimations() {
+        overridePendingTransition(AnimationConstants.FADE_IN, AnimationConstants.FADE_OUT)
```

Comment:
I figured we could use fade animations in all places where we were currently overriding pending transitions, as this results in a much smoother look. Any concerns about this?

Reply:
Does it change anything? This activity has no UI

Reply:
That one is indeed a bad example 🙈 There is a noticeable difference with the payment sheet though. I figured it wouldn’t hurt to apply the same everywhere to discourage using no animations.